### ISSUE TITLE
Implement Loss Functions 

### ISSUE THREAD

**Issue by farkas93** [farkas93/PatinAI#3]
Implement Loss functions to be independent of the Optimizers.
Currently hard-coded loss function.

**Comment by farkas93** [comment]
Implemented crossentropy, MSE, MAE and RMSE.
